crowbar_framework/app/controllers/api/upgrade_controller.rb

```diff
@@ -80,6 +80,12 @@ def cancel
     render json: { error: e.message }, status: :unprocessable_entity
   end
 
+  api :GET, "/api/upgrade/repocheck", "Check for missing node repositories"
```

Comment:
https://github.com/crowbar/crowbar-core/issues/652 adapts the response of the admin repocheck